[2024-11-10T13:02:44.141+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-11-10T13:02:44.173+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipeline.fetch_data_from_json manual__2024-11-10T13:02:20.567452+00:00 [queued]>
[2024-11-10T13:02:44.193+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipeline.fetch_data_from_json manual__2024-11-10T13:02:20.567452+00:00 [queued]>
[2024-11-10T13:02:44.199+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-11-10T13:02:44.220+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): fetch_data_from_json> on 2024-11-10 13:02:20.567452+00:00
[2024-11-10T13:02:44.227+0000] {standard_task_runner.py:64} INFO - Started process 221 to run task
[2024-11-10T13:02:44.231+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'pipeline', 'fetch_data_from_json', 'manual__2024-11-10T13:02:20.567452+00:00', '--job-id', '529', '--raw', '--subdir', 'DAGS_FOLDER/pipline.py', '--cfg-path', '/tmp/tmpum_06_91']
[2024-11-10T13:02:44.235+0000] {standard_task_runner.py:91} INFO - Job 529: Subtask fetch_data_from_json
[2024-11-10T13:02:44.283+0000] {task_command.py:426} INFO - Running <TaskInstance: pipeline.fetch_data_from_json manual__2024-11-10T13:02:20.567452+00:00 [running]> on host 781049d6fac8
[2024-11-10T13:02:44.375+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='pipeline' AIRFLOW_CTX_TASK_ID='fetch_data_from_json' AIRFLOW_CTX_EXECUTION_DATE='2024-11-10T13:02:20.567452+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-10T13:02:20.567452+00:00'
[2024-11-10T13:02:44.377+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-11-10T13:02:44.378+0000] {logging_mixin.py:188} INFO - Error reading JSON file: [Errno 2] No such file or directory: 'C:/Users/mohammed/Desktop/Airflow/Data-Engineering-Pipeline/***/data_set/journals.json'
[2024-11-10T13:02:44.378+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-11-10T13:02:44.379+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-11-10T13:02:44.388+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=pipeline, task_id=fetch_data_from_json, run_id=manual__2024-11-10T13:02:20.567452+00:00, execution_date=20241110T130220, start_date=20241110T130244, end_date=20241110T130244
[2024-11-10T13:02:44.443+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-11-10T13:02:44.473+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-11-10T13:02:44.479+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
